frontend/src/vars.ts

```diff
@@ -0,0 +1,2 @@
+// z-indexes from vars.scss
```

Comment:
So we can probably remove `:export { ... }` from `vars.scss`?

Reply:
🦔 💨